safe_transaction_service/history/services/index_service.py

```diff
@@ -88,6 +106,51 @@ def block_get_or_create_from_block_hash(self, block_hash: int):
                 block, confirmed=confirmed
             )
 
+    def get_indexing_status(self) -> IndexingStatus:
+        current_block_number = self.ethereum_client.current_block_number
+        erc20_block_number = SafeContract.objects.aggregate(
+            min_erc20_block_number=Min("erc20_block_number")
+        )["min_erc20_block_number"]
+        if erc20_block_number is None:  # Still nothing indexed
+            erc20_block_number = current_block_number
```

Comment:
I didn't get this.
`current_block_number` is last block indexed? and `erc20_block_number` is last indexed block, I'd said that the last block indexed should be max instead min, but I'm sure that I'm missing something.

Reply:
`current_block_number` is the current block on the chain, and `erc20_block_number` is the block where erc20 indexing is stuck, so, the minimum block number for all the Safes

Reply:
Ok, then `current_block_number` means the position where we are on chain, and for this reason
`if erc20_block_number is None -> erc20_block_number = current block.` that means that there wasn't anything to index until such `current_block_number`, true?